mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/helpers/boot.ts

```diff
@@ -0,0 +1,12 @@
+import { select } from '@wordpress/data';
+import { initializeApi, updateSection } from '../api';
+import { Section, storeName } from '../store';
+
+export function boot() {
+  initializeApi();
+  select(storeName)
+    .getSections()
+    .forEach((section: Section) => {
+      void updateSection(section);
+    });
```

Comment:
I already commented on this within the `updateSection`. I think it should be an action and in the "booting" process we should just call `dispatch`. Then, I guess, it doesn't even need its own file and method and could go directly to `index`.